src/Gir/Generation/Method.cs

```diff
@@ -8,9 +8,14 @@ public partial class Method : INativeCallable
 
 		public void Generate (IGeneratable parent, IndentWriter writer)
 		{
-			if (Deprecated)
-				writer.WriteLine ($"[Obsolete (\"(Version: {DeprecatedVersion}) {DocDeprecated.Text}\")]");
-
+			if (!string.IsNullOrEmpty (Deprecated)) {
```

Comment:
Which gobject-introspection version was used for these files? This might happen if the gi-scanner is too old and the gir format has changed.

Reply:
The version claimed to be the same gir format version